module/zfs/spa_stats.c

```diff
@@ -131,6 +131,7 @@ spa_read_history_init(spa_t *spa)
 	shl->procfs_list.pl_private = shl;
 	procfs_list_install(module,
 	    "reads",
+	    0644,
```

Comment:
Let's make this one 0600 too.  This isn't enabled by default, but when enabled we should only allow root access.  The same goes for `dbufs` which is registered in `dbuf_stats_hash_table_init()`,